.env.test

```diff
@@ -1,3 +1,4 @@
 CONSIDER_ALL_EMAIL_DOMAINS_AS_NON_FREE=False
 DO_NOT_USE_ANNUAIRE_EMAILS=False
 DO_NOT_CHECK_EMAIL_DELIVERABILITY=False
+DATABASE_URL=postgres://moncomptepro:moncomptepro@127.0.0.1:5432/moncomptepro
```

Comment:
@rdubigny This is temporally require to make "npm test" happy as it's importing the global config/env.ts